apps/backend/src/modules/email/email.service.ts

```
@@ -4,6 +4,8 @@
 import { PaymentSuccessEmail } from '../../emails/payment-success-email'
 import { PaymentFailedEmail } from '../../emails/payment-failed-email'
 import { SubscriptionCanceledEmail } from '../../emails/subscription-canceled-email'
+import type { ContactFormRequest } from '@repo/shared/types/domain'
+import type { ContactFormRequest } from '@repo/shared/types/domain'
```

Biome (2.1.2)

[error] Shouldn't redeclare 'ContactFormRequest'. Consider to delete it or rename it.

'ContactFormRequest' is defined here:


(lint/suspicious/noRedeclare)